critiquebrainz/db/users.py

```diff
@@ -0,0 +1,17 @@
+from critiquebrainz import db
+from critiquebrainz.db import exceptions as db_exceptions
+import sqlalchemy
+
+
+def getusers_id(usernames):
+    """Get MusicBrainz ids of users
+    Args:
+        usernames (list): MusicBrainz usernames.
+    """
+    with db.engine.connect() as connection:
+        req = sqlalchemy.text("""SELECT * FROM "user" WHERE display_name IN :users""")
+        result = connection.execute(req, users=tuple(usernames))
+        if not result:
+            return db_exceptions.NoDataFoundException("Can't find users")
```

Comment:
I don't see much point in having this exception. What if we want to find users that exist, given a list of MusicBrainz usernames, and just ignore missing ones? This is probably what we want for the moderator list feature, considering that it's possible that moderator might not have an associated user in the CritiqueBrainz database.

I would suggest two options:

1. Add an `ignore_missing` argument. If it's set to True then return a list with users that were found and skip ones that are missing. If it's set to `False`, then check if all users passed in the `usernames` argument were found, and if not then raise an exception (ideally with a list of missing users).
2. Ignore missing users by default, but document this behaviour.